internal/backendstore/backendstore.go

```diff
@@ -86,18 +87,38 @@ func (b *BackendStore) ToggleBackendActiveStatus(backendName string, active bool
 	}
 }
 
+func (b *BackendStore) GetBackendHealthStatus(backendName string) v1alpha1.HealthStatus {
+	b.mu.RLock()
+	defer b.mu.RUnlock()
+
+	if _, ok := b.s3Backends[backendName]; ok {
+		return b.s3Backends[backendName].health
+	}
+
+	return v1alpha1.HealthStatusUnhealthy
```

Comment:
Does it make sense to return unknown here?

Reply:
Yes :+1: